client/src/components/Admin/ui/Label.js

```diff
@@ -57,7 +59,7 @@ const Label = ({ id, label, tooltipTitle, href }) => {
             sx={{ textTransform: "none" }}
             rel="noopener"
             component={Link}
-            href={href}
+            href={combinedLink} // Use the combined URL here
```

Comment:
I think we need to update the check we're doing before we render this `Go To Link` button. Instead of just `href`, we should probably check for `handle`